spread.yaml

```diff
@@ -0,0 +1,38 @@
+project: snap-confine
+
+environment:
+    REUSE_PROJECT: $(echo $REUSE_PROJECT)
+    PATH: /snap/bin:$PATH
+
+backends:
+    linode:
+        key: $(echo $SPREAD_LINODE_KEY)
+        systems:
+            - ubuntu-16.04-64-grub
+            - ubuntu-16.04-32-grub
+
+path: /spread/snap-confine
+
+exclude:
+    - .git
+
+prepare: |
+    [ "$REUSE_PROJECT" != 1 ] || exit 0
+
```

Comment:
I don't know, sorry, I think it will have to be something like `if grep ubuntu /etc/os-release` and similar.